README.rst

```diff
@@ -1,64 +1,44 @@
-===============
-Getting started
-===============
-
-------------------------------------------------
-Install dependencies & start the service locally
-------------------------------------------------
-
-1. Enter the project folder:
-
-.. code-block:: console
-
-    $ cd flexprep
-
-2. Install packages
-
-.. code-block:: console
-
-    $ poetry install
-
-
-3. Run the job
-
-.. code-block:: console
-
-    $ poetry run python -m flexprep
-
--------------------------------
-Run the tests and quality tools
--------------------------------
-
-1. Run tests
-
-.. code-block:: console
-
-    $ poetry run pytest
-
-2. Run pylint
-
-.. code-block:: console
-
-    $ poetry run pylint flexprep
-
-
-3. Run mypy
-
-.. code-block:: console
-
-    $ poetry run mypy flexprep
-
-
-----------------------
-Generate documentation
-----------------------
-
-.. code-block:: console
-
-    $ poetry run sphinx-build doc doc/_build
-
-Then open the index.html file generated in *flexprep/build/sphinx/html*
-
-
-.. HINT::
-   All **poetry run** prefixes in the commands can be avoided if running them within the poetry shell
+# FlexPrep
+### Container which pre-processes IFS data for [Flexpart](https://www.flexpart.eu/) in the European Weather Cloud ([EWC](https://europeanweather.cloud/)).
+
+FlexPrep is part of an automated system triggered by product dissemination via [ECPDS](https://confluence.ecmwf.int/pages/viewpage.action?pageId=228871373) using [Aviso](https://confluence.ecmwf.int/display/EWCLOUDKB/Aviso+Notification+System+on+EWC). This setup ensures that data processing is initiated automatically when new data is available. For detailed instructions on configuring and setting up Aviso for this system, please refer to the [aviso folder in this repository.](https://github.com/MeteoSwiss-APN/flexprep/blob/main/aviso/README.md).
+
+
+## Setup Instructions for VM on EWC
```

Comment:
Perhaps there should also be a section describing how to do local dev and testing if that's possible?

Reply:
Here the issue created so that I address it later: https://github.com/MeteoSwiss-APN/flexprep/issues/11